.github/workflows/posix.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         pattern: ["not single_cpu", "single_cpu"]
         # Don't test pyarrow v2/3: Causes timeouts in read_csv engine
         # even if tests are skipped/xfailed
-        pyarrow_version: ["5", "6", "7"]
+        pyarrow_version: ["5", "7"]
```

Comment:
does this merit a comment about why 6 is skipped?  the "v2/3" comment may be out of date?

Reply:
IMO not really. We could very well be testing 6, but the intention is just to reduce CI jobs.

And I think the v2/3 comment is still warranted. A workaround was never discovered that allowed us to test pyarrow 2 or 3 without the CI stalling & failing.